### DISCUSSION TITLE
Checkbox styling in Rich Text Editor tasklist

### DISCUSSION THREAD

**Discussion by sindrefl** [mantinedev/mantine#6974] [Q&A]
Hello,

If custom styling (ie css modules in "classnames" props/styles api defaults) or props (ie variant="outline") are applied throught the MantineProvider to the Checkbox component, then these stylings are not applied to the Checkbox in the Tasklist in the @mantine/tiptap package (example https://mantine.dev/x/tiptap/#tasks).
This makes sense from an architectural standpoint as the Rich Text Editor checkbox list does not contain React components, but plain html in a TypographyStylesProvider.

What would be the best way to achieve that these style overrides propagate to the tasklist?
And could this be considered a bug in the mantine library as it would be logical to think that the styles should be propagated?

**Comment by rtivital** [comment]
To change styles of checkboxes, use Styles API and follow the same selectors that are used in the source code – https://github.com/mantinedev/mantine/blob/master/packages/@mantine/tiptap/src/RichTextEditor.module.css/#L305-L369
